### PR TITLE
Updatedb2

### PR DESCRIPTION
**Pull request by clukhei**
- Features include:
1. File Upload page 
2. File input 
3. Confirm file upload
4. Progress bar for file load
5. Successful Upload : 
- Success page
- Success Tables/ Tabs
- Pagination for Tables
- Summary data of upload
6. Failed Upload 

** using localhost API

### CODE REVIEW

File: src/components/UpdateDB/index.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+          )}
+        </Box>
+      )}
+    </div>
+  )
+}
+
+const UploadMsg = () => {
+  return (
+    <div>
```
Comment (SheldonCheng) [inline src/components/UpdateDB/index.js]: Give this div a `display: flex`

File: src/components/UpdateDB/index.css [hunk 2]
```diff
@@ -0,0 +1,255 @@
+.updatedb-box {
+  width: 62.1%;
```
Comment (SheldonCheng) [inline src/components/UpdateDB/index.css]: don't set width here

File: src/components/UpdateDB/Success.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      </div>
+    </div>
+  )
+}
+
+export default SuccessUpload
+
+const Table = props => {
+  return (
+    <table class="table">
```
Comment (SheldonCheng) [inline src/components/UpdateDB/Success.js]: change to `className`

File: src/components/UpdateDB/Success.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+      </tbody>
+    </table>
+  )
+}
+
+function ListItem(props) {
+  const listElements = props.data
+  const listComponents = listElements.map(item => {
+    return (
+      <tr className="d-flex ">
```
Comment (SheldonCheng) [inline src/components/UpdateDB/Success.js]: missing `key` property

File: src/components/UpdateDB/index.js (hunk 1 above)
Comment (clukhei) [reply src/components/UpdateDB/index.js]: I added div with display:flex to the line above 101 in this image instead. The uploadMsg div is only for the words in the upload file page. 
image

**Pull request merged**